### DISCUSSION TITLE
The order of components in files

### DISCUSSION THREAD

**Discussion by biglinn** [SSCHAcode/python-sscha#343] [Q&A] [answered]
Hi, @mesonepigreco:
Are orders of components in three files(scf_population1_XX.dat, u_population1_XX.dat and forces_population1_XX.dat) corresponding? 

Take KTaO3 supercell as an example, the order of components in "scf_population1_XX.dat"  is K Ta O O O, K Ta O O O, K Ta O O O...
![image](https://github.com/SSCHAcode/python-sscha/assets/155951912/7cb76ef4-3e9d-431f-beab-fa53697111e3)

So, does it mean that the same is true of u_population1_XX.dat and forces_population1_XX.dat?

**Comment by mesonepigreco** [comment, marked as answer]
Yes, the order is consistent among all the files.
